### PR TITLE
pyfaf: Add action for deleting all unassigned builds

### PR DESCRIPTION
**Pull request by marusak**
In commit 8b9bcac was introduced assigning releases to build. Also in
a6b9ba4 was introduced cleaning of builds (packages) for specific
assigned release. However, unassigned builds cannot be removed.

This commit introduces action for cleaning such builds (packages). Since
I understand this is rather a dangerous action, I made it run by default
as dry run and only by using `--force` packages can be removed.

Signed-off-by: Matej Marusak <mmarusak@redhat.com>

### CODE REVIEW

**Comment by marusak** [comment]
retest this please

**Comment by marusak** [comment]
I should write tests for this, marking WIP

File: tests/actions [hunk 1]
```diff
@@ -379,6 +379,56 @@ class ActionsTestCase(faftests.DatabaseCase):
...
+
+        # add package and lob that will not be deleted
+        pkg_stay = Package()
+        pkg_stay.build = self.db.session.query(Build).first()
+        pkg_stay.arch = self.db.session.query(Arch).first()
+        pkg_stay.name = "pkg-test-stay"
+        self.db.session.add(pkg_stay)
+        self.db.session.flush()
+
+        config["storage.lobdir"] = "/tmp/faf_test_data/lob"
```
Comment (xsuchy) [inline tests/actions]: Can you  replace this (and the other occurence) with `tempfile.mkdtemp(prefix="faf")`?

Comment (marusak) [inline tests/actions]: Sure. Done. I removed the second occurrence since it was useless there.

**Pull request merged**